index.html

```diff
@@ -123,21 +131,130 @@
+      .domain(d3.extent(data, function (a) {return a.value; }))
       .range([height, 0])
       .nice();
 
     // AXES
 
+
+     var xAxis = d3.svg.axis()
+                   .scale(x)
+                   .orient("bottom");
+ 
+     svg.append("g")
+        .attr("class", "x axis")
+        .attr("transform", "translate(0," + (height) + ")")
+        .call(xAxis)
+       
+        .append("text")
+        .attr("x", 1998)
+        .attr("y", 2015)
+        .attr("dy", ".9em")
+        .style("text-anchor", "start")
+        .text("Year");
+ 
+     var yAxis = d3.svg.axis()
+                   .scale(y)
+                   .tickSize(width)
+                   .orient("right")
+                   .ticks(8);
+ 
+     svg.append("g")
+        .attr("class", "y axis")
+        .attr("transform","translate(0,0)")
+        .call(yAxis)
+ 
+        .append("text")
+        .attr("transform", "rotate(-90)")
+        .attr("x", 666)
+        .attr("y", 8700000000)
+        .attr("dy", ".9em")
+        .style("text-anchor", "start")
+        .text("Amount of Incentive");
+ 
+       var gy = svg.append("g")
+                   .attr("class", "y axis")
+                   .call(yAxis);
+ 
+       gy.selectAll("g").filter(function(a) {return a;})
+         .classed("minor", true);
+ 
+       gy.selectAll("text")
+         .attr("x", 666)
+         .attr("dy", 8700000000);
+
+      var formatNumber = d3.format(".0f");
+  
+      var xAxis = d3.svg.axis()
+                    .scale(x)
+                    .tickFormat(formatCurrency)
+                    .tickSize(height)
+                    .orient("bottom"); 
+  
+      svg.append("g")
+         .attr("class", "x axis")
+         //.attr("transform", "translate(0," + (height) + ")")
+         .call(xAxis)
+        
+         .append("text")
+         .attr("x", 250)
+         .attr("y", 580)
+         .style("text-anchor", "start")
+         .text("Year");
+  
+      var gx = svg.append("g")
+                  .attr("class", "x axis")
+                  .call(xAxis);
+  
+      gx.selectAll("g").filter(function(a) {return a;})
+        .classed("minor", true);
+  
+      var yAxis = d3.svg.axis()
+                    .scale(y)
+                    .tickSize(width)
+                    .orient("right");
+  
+      svg.append("g")
+         .attr("class", "y axis")
+         .attr("transform","translate(0,0)")
+         .call(yAxis)
+  
+         .append("text")
+         .attr("transform", "rotate(-90)")
+         .attr("x", -300)
+         .attr("y", -60)
+         .attr("dy", ".9em")
+         .style("text-anchor", "start")
+         .text("");
+  
+      var gy = svg.append("g")
+                  .attr("class", "y axis")
+                  .call(yAxis);
+  
+        gy.selectAll("g").filter(function(a) {return a;})
+          .classed("minor", true);
+  
+        gy.selectAll("text")
+          .attr("x", -30)
+          .attr("dy", -4);
```

Comment:
If I'm counting correctly, you've got six total axes here.

Your axes are reproducing like bunnies. 🐰 

I'm getting the sense that you've copied a bunch of code in at various points. I think it's key to move incrementally and understand each piece that you're bringing in. So, for example, instead of copying the entirety of the example block where the lines stretched all the way across, you might try copying individual lines to see what they do. What does `.tickSize()` do? Try adding that to your existing axis and see what changes.